src/bus/mod.rs

```diff
@@ -431,14 +424,14 @@ mod test {
 	fn test_static_buffer() {
 		let buffer1 = static_buffer!(128);
 		assert!(buffer1.len() == 128);
-		for i in 0..buffer1.len() {
-			assert!(buffer1[i] == 0);
+		for e in buffer1 {
+			assert!(*e == 0);
```

Comment:
```suggestion
		for i in 0..buffer1.len() {
			assert!(buffer1[i] == 0, "i: {i}");
```

There is some value in keeping the original if we also log `i` in the assertion. It will show a clearer error message on failure.